test_fms/time_manager/test_time_manager.F90

```diff
@@ -603,9 +603,58 @@ program test_time_manager
     endif
   endif
  !==============================================================================================
-  write(outunit,'(/,a)') '############################################################################'
-  write(outunit,'(a,i6)') ' ticks_per_second=',get_ticks_per_second()
+
+  if(test20) then
+     write(outunit,'(/,a)') '#################################  test20  #################################'
```

Comment:
Please add a description to what test20 is testing.  We really need to get away from test numbers, and more to test descriptions.

Reply:
@underwoo  thank you for the review, I'll make sure all the changes are made when the PR is ready!